package.json

```diff
@@ -34,11 +34,19 @@
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
     "@testing-library/react": "^9.1.3",
+    "@wdio/allure-reporter": "^5.7.11",
+    "@wdio/cli": "^5.0.0",
+    "@wdio/junit-reporter": "^5.0.0",
+    "@wdio/local-runner": "^5.0.0",
+    "@wdio/mocha-framework": "^5.0.0",
+    "@wdio/selenium-standalone-service": "^5.0.0",
+    "@wdio/spec-reporter": "^5.0.0",
     "auto": "^7.8.0",
     "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
     "babel-plugin-add-module-exports": "^1.0.2",
     "babel-plugin-rewire": "^1.2.0",
+    "chai": "^4.0",
```

Comment:
Is this coming from some other package or being added as a new dependency?

Reply:
My bad - these shouldn't be here. Let me clean up the PR.